src/WebApp/Extensions/Extensions.cs

```diff
@@ -91,17 +94,22 @@ public static void AddAuthenticationServices(this IHostApplicationBuilder builde
     private static void AddAIServices(this IHostApplicationBuilder builder)
     {
         var openAIOptions = builder.Configuration.GetSection("AI").Get<AIOptions>()?.OpenAI;
-        if (!string.IsNullOrWhiteSpace(openAIOptions?.ApiKey))
+        var deploymentName = openAIOptions?.ChatModel;
+
+        if (!string.IsNullOrWhiteSpace(deploymentName))
         {
-            var kernelBuilder = builder.Services.AddKernel();
-            if (!string.IsNullOrWhiteSpace(openAIOptions.Endpoint))
-            {
-                kernelBuilder.AddAzureOpenAIChatCompletion(openAIOptions.ChatModel, openAIOptions.Endpoint, openAIOptions.ApiKey);
-            }
-            else
+            builder.Services.AddKernel();
+            builder.AddAzureOpenAI("chatOpenAi");
+
+            // Note: SemanticKernel should expose an extension method AddAzureOpenAIChatCompletion() that does the following.
```

Comment:
Maybe there is already an extension for this.

Reply:
There isn't currently. I wanted us to wait to add one until the aspire component was on the way such that it'd be more common to have one of these available.

Reply:
https://github.com/microsoft/semantic-kernel/pull/4555